components/hamburger-menu.vue

```diff
@@ -63,10 +63,10 @@
     "translate": "原神 自動翻訳"
   },
   "zh-CN": {
-    "about": "About",
-    "opendata": "Opendata·API",
-    "history": "History",
-    "translate": "Genshin Machine Translate (Japanese)"
+    "about": "关于",
+    "opendata": "开放数据 · API",
```

Comment:
Just a confirmation. Is it common in Chinese to add whitespaces around dot (·)？

Reply:
Well, both is OK.

Reply:
OK, then let's keep as is.